packages/build-info/src/node/bin.ts

```diff
@@ -20,26 +22,31 @@ yargs(hideBin(argv))
           string: true,
           describe: `The root directory of the project if different from projectDir`,
         },
-        featureFlags: {
+        devCycleUser: {
           string: true,
-          describe: 'comma separated list of feature flags',
-          coerce: (value = '') =>
-            value
-              .split(',')
-              .map((flag) => flag.trim())
-              .filter((flag) => flag.length)
-              .reduce((prev, cur) => ({ ...prev, [cur]: true }), {}),
+          default: '{}',
+          describe: 'DevCycle user that is used for the feature flag client',
+          coerce: (value) => {
+            try {
+              return JSON.parse(value)
+            } catch {
+              return {}
+            }
+          },
         },
       }),
-    async ({ projectDir, rootDir, featureFlags }: Args) => {
+    async ({ projectDir, rootDir, devCycleUser }: Args) => {
       // start bugsnag reporting
       await initializeMetrics()
+      const featureFlags = await getFeatureFlags(devCycleUser)
```

Comment:
With this change, will we not then fully depend on devCycle? I'm curious how this will be backwards compatible for the time being?

Reply:
:) amazing, okido! Thanks!